config/migrate.ts

```diff
@@ -159,6 +159,15 @@ export function mergeConfigProperties(
     }
   });
 
+  if (globalConfig.flags || deprecatedConfig.flags) {
```

Comment:
Should we handle conflicts (like, push them to the conflicts array)? Alternatively, since global config is newer maybe it would make sense to let any global config flags override the deprecated config flags?

Reply:
Makes sense to me. Defaulting to the new config settings seems fine